packages/mds-event-processor/src/utils.ts

```diff
@@ -30,7 +30,7 @@ export async function getTripId(deviceState: StateEntry): Promise<string | null>
   const { provider_id, device_id, timestamp } = deviceState
   const tripsEvents = await cache.readTripsEvents(`${provider_id}:${device_id}`)
   if (!tripsEvents) {
-    await log.warn('NO PRIOR TRIP EVENTS FOUND')
+    log.info(`NO PRIOR TRIP EVENTS FOUND for ${provider_id}:${device_id}`)
```

Comment:
Typically we await `log` methods; they are async. I am surprised you didn't get a linter warning.

Reply:
The 'info' method is synchronous

Reply:
That's right. I remember that now. We should probably make it async for consistency.